.github/workflows/main.yml

```diff
@@ -56,24 +12,37 @@ jobs:
         # Github Actions requires a single row to be added to the build matrix.
         # See https://help.github.com/en/articles/workflow-syntax-for-github-actions.
         name: [
```

Comment:
do we need to add more fields here? 
I read "You cannot use include to add new jobs to a build matrix." from https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#example-including-configurations-in-a-matrix-build

Reply:
but macos and linux build do appear, weird

Reply:
Huh, interesting. It's clearly building all 3 OSes so I think we might not need `name` here at all, but I'll check, thanks for pointing it out. A little bit confused what that sentence is saying haha.